src/commands/dev.ts

```diff
@@ -69,6 +69,7 @@ const command = defineCommand({
       // Fork nuxt dev process
       const devProxy = await _createDevProxy(nuxtOptions, listenOptions)
       await _startSubprocess(devProxy, ctx.rawArgs)
+      return devProxy?.listener?.url
```

Comment:
Same as citty, we might return `{ listener }` here too (also notice when using a public URL or domain or tunnel, there are multiple listener URLs as well. `url` is the _prefered_ one we usually show first and used for public/qr when available)

Reply:
@pi0 I pushed a new commit with the change you requested! Thanks!